docs/build-tools/overview.md

```diff
@@ -32,19 +36,21 @@ compiler plugin and `-Yrangepos` option enabled.
 
 ## Goto library dependencies
 
-**Automatic**: it is possible to navigate Scala+Java library dependencies using "Goto
-definition".
+**Automatic**: it is possible to navigate Scala+Java library dependencies using
+"Goto definition".
 
-**Semi-automatic**: navigation in library dependency sources works as
-long as the
+**Semi-automatic**: navigation in library dependency sources works as long as
+the
 [Bloop JSON files](https://scalacenter.github.io/bloop/docs/configuration-format/)
 are populated with `*-sources.jar`.
 
 ## Find references
 
 **Automatic**: it is possible to find all references to a symbol in the project
 
-**Semi-automatic**: it is possible to 'Find symbol references' as soon the SemanticDB compiler plugin is manually enabled in the build, check separate build tool pages for details.
+**Semi-automatic**: it is possible to 'Find symbol references' as soon the
+SemanticDB compiler plugin is manually enabled in the build, check separate
```

Comment:
Super nitpicky. but the comma should be a period and `Check separate build tool pages for details.` should be its own sentence.

Reply:
Can you open a separate PR? This paragraph is unchanged in this PR, only formatting changes.